.changeset/serious-dingos-begin.md

```diff
@@ -0,0 +1,5 @@
+---
+"@azure-tools/cadl-ranch-specs": minor
```

Comment:
a `patch` instead of `minor` release would be enough?

Reply:
indeed